libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -377,11 +369,9 @@ public void onProgressChange(Location location, RouteProgress routeProgress) {
   private OffRouteListener offRouteListener = new OffRouteListener() {
     @Override
     public void userOffRoute(Location location) {
-      if (hasNetworkConnection()) {
```

Comment:
Is the `offRouteListener` only getting fired now when the device is online? This doesn't cause any issues?

Reply:
Connectivity checks are now done in `NavigationViewRouter` with the `ConnectivityStatusProvider`, so this check was no longer necessary / would block the new logic.